src/software/abinit.jl

```diff
@@ -962,3 +962,29 @@ function write_abinit_modes(modes::Array{SVector{6,Float64}}, energies::Vector{F
         end
     end
 end
+
+"""
+    read_abinit_anaddb_PHDOS(filename::AbstractString) -> Tuple{DensityOfStates, Vector{ProjectedDensityOfStates}}
+Reads the PHDOS file from ABINIT's anaddb script.
+"""
+function read_abinit_anaddb_PHDOS(filename::AbstractString)
+    data = readlines(filename)
+    # Skip header (lines 1-8)
+    data = data[9:length(data)]
+    # Initialize matrix to store data
+    # Energy | total PHDOS | int PHDOS | AtomType 1 PHDOS | AtomType 1 intPHDOS | ...
+    num_col = length(split(data[1]))
+    data_new = Matrix{Float64}(undef,length(data),num_col)
+    # Read in data
+    for i in 1:length(data)
```

Comment:
Avoid indexing with `1:length(A)` - instead, use `eachindex(A)` (or `axes(A, d)` for multidimensional arrays where you want to loop through a particular dimension). Although this is an uncommon use case, sometimes, people will want to use data structures that do not have indices starting at 1. In this case, I don't think it'll make a difference, but it's just better practice.